smbus2/smbus2.py

```diff
@@ -658,3 +658,31 @@ def i2c_rdwr(self, *i2c_msgs):
         """
         ioctl_data = i2c_rdwr_ioctl_data.create(*i2c_msgs)
         ioctl(self.fd, I2C_RDWR, ioctl_data)
+
+    def i2c_rd(self, i2c_addr, length):
+        """
+        Perform a single i2c read operation, given an i2c_addr and length.
+
+        :param i2c_addr: i2c address for the read operation.
+        :type i2c_addr: int
+        :param length: length of read.
+        :type length: int
+        :rtype: list
+        """
+        msg = i2c_msg.read(i2c_addr, length)
+        self.i2c_rdwr(msg)
+        return list(msg)
+
+    def i2c_wr(self, i2c_addr, buf):
+        """
+        Perform a single i2c write operation, given an i2c_addr and a
+        buffer to copy.
+
+        :param i2c_addr: i2c address for the write operation.
+        :type i2c_addr: int
+        :param buf: buffer to write.
+        :type buf: list
```

Comment:
Should be `:type buf: list or str` as it takes both. Yeah, I realize that the docstring for `i2c_msg.write` doesn't say that, but that one also should.